tutorials/plugins/editor/making_main_screen_plugins.rst

```diff
@@ -66,176 +47,92 @@ file. In our example, ``main_screen_plugin.gd``.
        pass
 
 
+    func handles(obj):
```

Comment:
This makes it sound as if `handles` is a required method for main screen plugins, and it is not. This subject is better detailed in a separate section of this article, otherwise you fall into the same issue as with the old article and side panels.

Reply:
Good point. I changed the examples to just a mention, but also users can always download the demo project if they want to see an example of how `handles` works.